cost/turbonomics/scale_virtual_machines_recommendations/aws/README.md

```diff
@@ -0,0 +1,21 @@
+# Turbonomics Scale Virtual Machines Recommendations AWS
+
+## What it does
+
+The Turbonomics Scale Virtual Machines Recommendations AWS policy utilizes Turbonomics [Credentials](https://turbonomic.com/api/v3/markets/{market_uuid}/actions) endpoint to provide Scale Virtual Machines Recommendations. From these recommendations we provide monthly savings estimates based on Turbonomics per hour costs.
```

Comment:
Since this URL doesn't exist, maybe you should put it as text instead of URL, otherwise you'll get always issues with textlint.

Reply:
Done, thank you!